### ISSUE TITLE
Is this library constant time?

### ISSUE THREAD

**Issue by FredericJacobs** [tonyg/js-nacl#30]

**Comment by elimisteve** [comment]
Yes, please include a prominent warning in the README that says that this crypto has not been thoroughly audited; that's exactly what I came here to check because it's necessary for building systems that are secure!
